### ISSUE TITLE
Can the 0.3.0 package be referenced on npm ?

### ISSUE THREAD

**Issue by VincentCATILLON** [zoontek/react-native-permissions#99]
Hi,

I'd like to use the "openSettings" method added in 0.3.0 version: #82
Can you add the 0.3.0 version with this method support on [npmjs](https://www.npmjs.com/package/react-native-permissions) ?

Thx

**Comment by ptfly** [comment]
Yes please ;)

**Comment by VincentCATILLON** [comment]
You can use it like that in your package.json for the moment:

```
"react-native-permissions": "github:yonahforst/react-native-permissions.git#f54d001cbc476c8f2c12d9de90bc1c6af5973e6d",
```

**Comment by yonahforst** [comment]
@VincentCATILLON  - I've just published version 1.0.0 to npm. (unfortunately, that version also removes `openSettings()` from the Android API. 

(You can try https://github.com/Aleksandern/react-native-android-settings-library)
